packaging_automation/requirements.in

```diff
@@ -1,17 +1,16 @@
-wheel
 docker
 GitPython
 Jinja2
+parameters_validation
 pathlib2
+psycopg2
 pycurl
 PyGithub
 pytest
-python-string-utils
-requests
-urllib3
 python-gnupg
-parameters_validation
+python-string-utils
```

Comment:
I see that order has changed. has hhanging the order affected the versions of libraries in requrements.txt?

Reply:
No.

If you check the order of my commits, you will see that:
1. I ran `pip-compile --upgrade` to bump package versions,
2. Sort the dependencies in `requirements.in` manually, and remove extra blank lines and spaces.
3. Create `.python-version` to store the supported python version alongside the code.

When I run `pip-compile --upgrade` again after sorting the entries in `requirements.in`, `requirements.txt` file remains unchanged.